packages/commonwealth/client/scripts/helpers/chain.ts

```diff
@@ -143,6 +143,12 @@ export const selectChain = async (chain?: ChainInfo): Promise<boolean> => {
       )
     ).default;
     newChain = new ERC721(chain, app);
+  } else if (chain.network === ChainNetwork.ERC1155) {
```

Comment:
I dont believe this will be required - we are not planing to require chains wishing to use 1155 gating be a ERC1155 specific community

Reply:
Will remove this

Reply:
Removed this